Best practices/Enzyme.md

```diff
@@ -78,3 +100,21 @@ expect(myComponent.find(ChildComponent)).toHaveLength(1);
 // over:
 expect(myComponent.find(ChildComponent).exists()).toBe(true);
 ```
+
+## Event simulations
+
+* When triggering events, trigger the callback function for the event, whenever possible.
```

Comment:
I think you can combine these into one rule; when triggering props on a composite component, use `trigger`; when doing it for native DOM events, use `simulate`

Reply:
You don't need to construct a valid mock event object